### ISSUE TITLE
[Bug]: SVG conversion back and forth.

### ISSUE THREAD

**Issue by ampersarnie** [bigbite/build-tools#34]
### What happened?

@g-elwell had mentioned on #33 during testing of that PR about conversion of SVGs going from to a string and then back to a component. We should investigate this and look into impacts and changes that are required. Gareth has gave reference to [this as config](https://react-svgr.com/docs/webpack/#use-svgr-and-asset-svg-in-the-same-project) we're likely to require.

### Which environments are you experiencing the issue on?

CI (CircleCI, Travis, etc), CLI (macOS, Windows, etc)

**Comment by ampersarnie** [comment]
Thanks for the expansive report on this @g-elwell. I definitely think it's worth making the change. It seems to me that we need to get this change in sooner rather than later as this is a breaking change that we already know about and will need to make, so best do it now. Can you open up a PR with your branch so I can review and assess from there? I'd also take a look at the Integration/Example Site that I added as part of #41 and add the example there also, just so we can keep up with integration testing.

**Comment by g-elwell** [comment]
@ampersarnie I've created #45 and checked out the example site. The SVG import in the example site uses the new syntax we're proposing so doesn't need to be updated. Is the example site just for reference or is there a way to build and run tests on it?

**Comment by ampersarnie** [comment]
@g-elwell The example site is there both for reference and to ensure we've got some basic integration testing. It runs in CI and does a build with the main commands, so if the build fails we know. It'll be getting expanded into something fuller as soon as possible really.

**Comment by g-elwell** [comment]
@ampersarnie Got it thanks and happy to help with any of that if needed. The SVG import in the example site will produce a runtime error but it will be fixed if/when #45 gets merged. Otherwise [this line](https://github.com/bigbite/build-tools/blob/bc98eedf04851bbb705f20a8a6b40af82100da1b/example-site/plugins/test-plugin/src/components/DisplayComponent.js#L1) would need to be changed to:
```js
import { ReactComponent as LogoSVG } from '../static/logo.svg';
```
